azure_arc_k8s_jumpstart/docs/aro_terraform.md

```diff
@@ -0,0 +1,232 @@
+# Overview
```

Comment:
change the file name. still showing terraform

Reply:
Done